samples/system-test/dlp_inspect_text_file_test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const execa = require('execa');
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+
+const REGION_TAG = 'dlp_inspect_text_file';
+const PROJECT_ID = process.env.GOOGLE_CLOUD_PROJECT;
```

Comment:
Same deal, I'd prefer to get this from `dlp.auth.getProjectId()`.  We should never require that env var to be set.

Reply:
All the existing DLP samples/tests rely on the environment variable, based on https://github.com/googleapis/nodejs-dlp/pull/154#discussion_r262211255 I'm not really wanting to refactor this in this PR. I will bring the idea of "project ID getter helpers" with the working group, perhaps we'll add some guidance on that.

Reply:
Please, please, please don't use the env variable. Users should never use it like that. @JustinBeckwith , fyi, you can just use `dlp.getProjectId()`, no need to use the `auth` property.

We can't show patters that we definitely don't want in the `sample format`.

Reply:
1) I am not trying to make any serious decisions or recommendations about how things should be done in the tests. Yes, the line in the test you're commenting on uses an env var, which is already the status quo for all the sample tests in this repo. Very soon, concern about the tests and how they're structured will be minimized as they will be auto-generated by the tool that auto-generates the samples. I created three new test files because I'm assuming that the tests for each sample will be auto-generated as separate test files.

2) We are not optimizing for the experience of users looking at our test files, the primary concern of this PR is what the code will look like on [cloud.google.com](https://cloud.google.com), which in this PR is proposed to be:

```js
/**
 * TODO(developer): Uncomment these variables before running the sample.
 */
// const projectId = 'YOUR_PROJECT_ID';
// const filepath = 'path/to/file.txt';

// Imports the Google Cloud Data Loss Prevention library
const DLP = require('@google-cloud/dlp');

// Import other required libraries
const fs = require('fs');
const util = require('util');
const readFile = util.promisify(fs.readFile);

// Instantiates a client
const dlp = new DLP.DlpServiceClient();

async function inspectTextFile() {
  // Get the bytes of the file
  const fileBytes = Buffer.from(await readFile(filepath)).toString('base64');

  // Construct request
  const request = {
    parent: dlp.projectPath(projectId),
    item: {
      byteItem: {
        type: 'TEXT_UTF8',
        data: fileBytes,
      },
    },
    inspectConfig: {
      // The infoTypes of information to match
      infoTypes: [
        {name: 'PHONE_NUMBER'},
        {name: 'EMAIL_ADDRESS'},
        {name: 'CREDIT_CARD_NUMBER'},
      ],
      // Whether to include the matching string
      includeQuote: true,
    },
  };

  // Run request
  const [inspectResponse] = await dlp.inspectContent(request);
  const findings = inspectResponse.result.findings;
  console.log(`Findings: ${!findings.length ? 'None' : ''}`);
  for (const finding of findings) {
    console.log(`  Quote: ${finding.quote}`);
    console.log(`  Info type: ${finding.infoType.name}`);
    console.log(`  Likelihood: ${finding.likelihood}`);
  }
}

inspectTextFile(projectId, filepath);
```

3) This PR will never be merged. It will be used as template, eventually leading to another PR that replaces all the samples in this repo with auto-generated ones.

4) I don't agree that `dlp.getProjectId()` is the right recommendation, there's no guarantee that the calling project ID should or has to be the same as the project resource identified in the call.

Reply:
If I care as little about the tests as I claim, why not use `dlp.getProjectId()`?

Done.

Reply:
Thanks!